frontend/src/components/dashboard/Dashboard.vue

```diff
@@ -37,19 +37,23 @@ const userRoleAssignments = ref<FamApplicationUserRoleAssignmentGet[]>();
 
 onMounted(async () => {
     // Reload list each time we navigate to this page to avoid forcing user to refresh if their access changes.
-    try {
-        loading.value = true;
-        applicationsUserAdministers.value = (
-            await applicationsApi.getApplications()
-        ).data;
-        if (isApplicationSelected) {
-            getAppUserRoleAssignment();
```

Comment:
we don't need to use "await" and "then" at the same time, the error is just caused by we didn't wait for the api call to get user role assignment, just change to "await getAppUserRoleAssignment();"

Reply:
Thanks @MCatherine1994 . So, what are you suggesting, to keep the try/catch block with the `await` on  `getAppUserRoleAssignment()`?

Reply:
yeah, I think that will do the same as your code, you need to test it though

Reply:
I have changed as suggested.